### ISSUE TITLE
token usage in chat not showing

### ISSUE THREAD

**Issue by ERMarti** [stackblitz/bolt.new#3560]
### Describe the bug

Token usage was not being displayed in chat. this forced me to go into settings to determine token usage.

### Link to the Bolt URL that caused the error

https://bolt.new/~/sb1-ml9xeh

### Steps to reproduce

just look at prompt/ chat. no token count is present.

### Expected behavior

expected to see a count of tokens used/ tokens left.

### Platform

```
Browser name = Firefox
Full version = 115.0
Major version = 115
navigator.appName = Netscape
navigator.userAgent = Mozilla/5.0 (Macintosh; Intel Mac OS X 10.13; rv:109.0) Gecko/20100101 Firefox/115.0
performance.memory = undefined
Username = ERMarti
Chat ID = cb2325967bda
```

**Comment by endocytosis** [comment]
@ERMarti, 

Can you go to Settings > General and then toggle "Show token usage in chat" to on and see if this resolves the issue?

**Comment by ERMarti** [comment]
Yes. It’s toggled on. I switched on then off. Signed out then back in. Still nothing.

Sent from my iPhone

On Dec 2, 2024, at 1:37 PM, Brian ***@***.***> wrote:

﻿

@ERMarti<https://github.com/ERMarti>,

Can you go to Settings > General and then toggle "Show token usage in chat" to on and see if this resolves the issue?

**Comment by endocytosis** [comment]
Hi all, 

Appreciate the feedback, I'll pass this along to the team!
